holoviews/plotting/bokeh/plot.py

```diff
@@ -463,6 +445,51 @@ class CompositePlot(BokehPlot):
     to such a plot.
     """
 
+    sizing_mode = param.ObjectSelector(default=None, objects=[
+        'fixed', 'stretch_width', 'stretch_height', 'stretch_both',
+        'scale_width', 'scale_height', 'scale_both', None], doc="""
```

Comment:
I guess it is worth exposing the bokeh semantics even if users should rarely need to set this explicitly (in which case I would expect it to override everything else)

Reply:
As this is CompositePlot I see this is only for containers...